apps/condo/domains/organization/schema/SbbolAuth.test.js

```diff
@@ -0,0 +1,128 @@
+const { SbbolOrganization } = require('@condo/domains/organization/sbbol/synch')
+const { EXAMPLE_USER_INFO, SBBOL_IMPORT_NAME } = require('@condo/domains/organization/sbbol/common')
+const faker = require('faker')
+const { v4: uuid } = require('uuid')
+const { INN_LENGTH } = require('@condo/domains/organization/constants/common')
+const { getSchemaCtx } = require('@core/keystone/schema')
+const { updateTestUser, User: UserAPI } = require('@condo/domains/user/utils/testSchema')
+const { makeClientWithRegisteredOrganization } = require('@condo/domains/organization/utils/testSchema/Organization')
+const { updateTestOrganization, Organization: OrganizationAPI, OrganizationEmployee: OrganizationEmployeeAPI  } = require('@condo/domains/organization/utils/testSchema')
+const { makeLoggedInAdminClient } = require('@core/keystone/test.utils')
+
+
+const userInfo = () => {
+    const inn = faker.datatype.number({
+        min: Math.pow(10, INN_LENGTH - 1) + 1,
+        max: Math.pow(10, INN_LENGTH) - 1,
+    })
+    return {
+        ...EXAMPLE_USER_INFO,
+        inn,
+        phone_number: faker.phone.phoneNumber('+792########'),
+        email: faker.internet.email(),
+        HashOrgId: uuid(),
+        userGuid: uuid(),
+    }
+}
+
+const createSync = async (userInfo) => {
+    const { keystone } = await getSchemaCtx('User')
+    const Sync = new SbbolOrganization({ keystone, userInfo })
```

Comment:
camelCase

Reply:
instances of classes use to start with upper letters